hbase-endpoint/src/test/java/org/apache/hadoop/hbase/coprocessor/TestCoprocessorEndpointTracing.java

```diff
@@ -228,7 +228,7 @@ public void onError(Throwable error) {
 
   @Test
   public void traceSyncTableEndpointCall() throws Exception {
-    final Connection connection = connectionRule.getConnection().toConnection();
+    final Connection connection = connectionRule.getSyncConnection();
```

Comment:
Hmm, you like that better? I think we use those words in our public API because we had to maintain backward compatibility, hence keeping `getConnection` and not renaming it to `getSyncConnection`. For this test code, I thought explicit names would be better. I don't feel strongly, was just my option as I wrote the code.

Reply:
I also do not have very strong opinion on which one is better but if want to add the 'Sync' prefix, then we need to change a bunch of classes, like SyncTable, SyncAdmin, etc, and most of them are marked as IA.Public. So in practical, I suggest we just keep it as is, at least one style is better then two styles in our code base...

Reply:
@Apache9 : Updated per your recommendation.